packages/apps/plugins/plugin-layout/src/components/MainLayout.tsx

```diff
@@ -82,11 +82,9 @@ export const MainLayout = ({ fullscreen, showComplementarySidebar = true }: Main
                   <MenuIcon weight='light' className={getSize(4)} />
                 </Button>
 
-                <Surface role='navbar-start' limit={1} />
+                <Surface role='navbar-start' />
                 <div role='none' className='grow' />
-
-                {/* TODO(burdon): Too specific? status? contentinfo? */}
-                <Surface role='presence' limit={1} />
+                <Surface role='navbar-end' direction='inline-reverse' />
```

Comment:
Love it! Thank you ✨

Reply:
FYI I didn't end up hooking this up inside `Surface` yet but you get the idea. If there comes a time that you actually need this working then let me know and we can find where it should inside `Surface`.